Common/TableProducer/zdc-task-intercalib.cxx

```diff
@@ -0,0 +1,155 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \brief Task for ZDC tower inter-calibration
+/// \author chiara.oppedisano@cern.ch
+// Minimal example to run this task:
+// export OPTIONS="-b --configuration json://config.json --aod-file AO2D.root"
+//  o2-analysis-timestamp ${OPTIONS} |
+//  o2-analysis-event-selection ${OPTIONS} |
+//  o2-analysis-track-propagation ${OPTIONS} |
+//  o2-analysis-mm-zdc-task-intercalib ${OPTIONS}
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/CCDB/EventSelectionParams.h"
+#include "Common/CCDB/TriggerAliases.h"
+#include "Common/DataModel/Centrality.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/ZDCInterCalib.h"
+
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TMinuit.h"
```

Comment:
I guess you don't need TMinuit here, right?...

Reply:
Sure! It was a leftover...sorry...and thanks a lot